use_cases.md

```diff
@@ -272,7 +270,7 @@ subjects:
 roleRef:
   kind: ClusterRole
   name: admin
-  apiGroup: rbac.authorization.k8s.io
+  API groups: rbac.authorization.k8s.io
```

Comment:
`apiGroup` is not a typo: that' the YAML key name and should not be subject of typo review.

Reply:
@YashThakare please, fix this.